src/components/tree/AposTree.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="apos-tree" :class="{ 'apos-tree--nested': nested }">
+    <AposTreeHeader
+      :headers="spacingRow" :spacer-only="true"
+      @calculated="setWidths"
+    />
+    <AposTreeHeader :headers="data.headers" :col-widths="colWidths" />
+    <AposTreeRows
+      :rows="data.rows"
+      :headers="data.headers"
+      :col-widths="colWidths"
+      :level="1"
+      :nested="nested"
+      @busy="setBusy"
+      @update="update"
+      list-id="root"
+      :draggable="draggable"
+      :tree-id="treeId"
+    />
+  </div>
+</template>
+
+<script>
+import AposTreeHeader from './AposTreeHeader';
+import AposTreeRows from './AposTreeRows';
+import AposHelpers from '../../mixins/AposHelpersMixin';
+import treeData from './data';
+
+export default {
+  name: 'AposTree',
+  components: {
+    AposTreeHeader,
+    AposTreeRows
+  },
+  mixins: [AposHelpers],
+  data() {
+    return {
+      // Copy the `data` property to mutate with VueDraggable.
+      nested: false,
+      colWidths: null,
+      treeId: this.generateId()
+    };
+  },
+  computed: {
+    data() {
+      return treeData;
```

Comment:
I assume this gets replaced with a service when integrated?

Reply:
That's right.